docs/index.rst

```diff
@@ -0,0 +1,30 @@
+Welcome to the C-Star Documentation!
+====================================
+
+**C-Star** is a python package that orchestrates the execution of a complete start-to-finish run of ROMS-MARBL.
+
```

Comment:
I think I'd suggest something like

```
**C-Star** is a python package for setting up and running ocean model simulations, with a particular focus on marine carbon dioxide removal (mCDR) applications.
```

... we could add that its current capability is limited to ROMS-MARBL?

Reply:
Sounds good. Changed in `3cce721`.